guides/getting_started_hassio.rst

```diff
@@ -30,55 +28,43 @@ After that, wait a bit until it is installed (this can take a while). Click "Sta
-
-    If you use `Microsoft Edge <https://www.microsoft.com/edge>`_ or `Google Chrome <https://www.google.com/chrome>`_, you will be able to install the initial configuration by connecting your ESP device to the computer that you're using to view the ESPHome Dashboard.
-    *You need to access the ESPHome Dashboard over HTTPS for this to work. This is a requirement of browsers to access your ESP device to ensure that we write the correct data.*
-
-    If you use another browser, you will have to connect the ESP devices to the machine running the ESPHome Dashboard and Home Assistant.
-
-    If the serial port is not showing up, you might not have the required drivers installed. These drivers work for most ESP devices:
-
-      * CP2102 (square chip): `driver <https://www.silabs.com/products/development-tools/software/usb-to-uart-bridge-vcp-drivers>`__
-      * CH341: `driver <https://github.com/nodemcu/nodemcu-devkit/tree/master/Drivers>`__
-
 
 Dashboard Interface
 -------------------
 
 Assuming you created your first configuration file with the wizard, let's take a quick
 tour of the ESPHome Dashboard interface.
 
-.. figure:: images/dashboard.png
+.. figure:: images/dashboard_states.png
     :align: center
     :width: 95.0%
 
 On the front page you will see all configurations for nodes you created. For each file,
-there are three basic actions you can perform:
+there are a few basic actions you can perform:
+
+- **UPDATE**: This button will show up if the device has not been installed with the same ESPHome version currently running.
+
+- **EDIT**: This will open the configuration editor.
 
-- **INSTALL**: This compiles the firmware for your node and installs it. Installation happens wirelessy if you have enabled the :doc:`/components/ota` in your configuration. Alternatively you can install it from your browser or via a device connected to the machine running the ESPHome Dashboard.
+- **LOGS**: With this command you can view all the logs the node is outputting. If a USB device is
+  connected, you can choose to use the serial connection. Otherwise it will use the built-in OTA logs.
 
-- **SHOW LOGS**: With this command you can view all the logs the node is outputting. If a USB device is
-  connected, it will attempt to use the serial connection. Otherwise it will use the built-in OTA logs.
+- **Overflow menu**: This is a dropdown menu which allows you to perform more actions.
 
-- **COMPILE**: This command compiles the firmware and gives you the option of downloading the generated
-  binary so that you can install it yourself from your computer using :ref:`ESPHome-flasher <esphome-flasher>`.
+  - **Validate**: This will validate the configuration file.
+  - **Install**: Opens the Install dialog.
+  - **Clean Build Files**: This will delete all of the generated build files and is a safe operation ot perform. Sometimes old files are not removed and can cause compile issues later on.
```

Comment:
I think pointing out that this is a safe operation is a good idea. I would be good to point out Delete below is not a safe operation. Unless there's a confirmation dialog in the UI that already does that.

Reply:
```suggestion
  - **Clean Build Files**: This will delete all of the generated build files and is a safe operation to perform. Sometimes old files are not removed and can cause compile issues later on.
```

Reply:
Good point about the delete